release-tools/desktop-cluster/stop-cluster.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# This script stops the local Elasticsearch and Kibana created by the start-cluster.sh script. It looks for the elasticsearch.pid and kibana.pid files in a given directory and kills the corresponding processes.
+
+set -e
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 /path/to/workdir"
+fi
+DIR=$1
+
+if [ -e $DIR/elasticsearch.pid ]; then
+    pid=$(<$DIR/elasticsearch.pid)
+    echo "Killing Elasticsearch (pid $pid)"
+    kill $pid
+    rm $DIR/elasticsearch.pid
```

Comment:
I don't see removing the $WORK_DIR anywhere after killing the processes (stopping the clusters). If everytime a new directory is created, should we add a step to remove the earlier one? Not sure if it is necessary as it is anyway created in /tmp

Reply:
Right now the log files are stored in $WORK_DIR, so I don't want to delete it automatically. Eventually this should change - we should be able to specify a log directory and have it automatically delete $WORK_DIR so you don't wind up with gigabytes of files in /tmp. That's a future enhancement.